10-General.inc.md

```diff
@@ -101,9 +103,9 @@ The structure of content protection related information in the CMAF containers u
 
 Note: This document uses the `cenc:` prefix to reference the XML namespace `urn:mpeg:cenc:2013` [[!CENC]].
 
-Initialization segments SHOULD NOT contain any `moov/pssh` box ([[!CMAF]] 7.4.3) and DASH clients MAY ignore such boxes when encountered. Instead, `pssh` boxes required for [=DRM system=] initialization are part of the [=DRM system configuration=] and SHOULD be placed in the MPD as `cenc:pssh` elements in [=DRM system=] specific `ContentProtection` descriptors.
+Initialization segments SHOULD NOT contain any `moov/pssh` box ([[!CMAF]], section 7.4.3) and DASH clients MAY ignore such boxes when encountered. Instead, `pssh` boxes required for [=DRM system=] initialization are part of the [=DRM system configuration=] and SHOULD be placed in the MPD as `cenc:pssh` elements in [=DRM system=] specific `ContentProtection` descriptors.
 
-Note: Placing the `pssh` boxes in the MPD has become common for purposes of operational agility - it is often easier to update MPD files than rewrite initialization segments when the default [=DRM system configuration=] needs to be updated. Furthermore, in some scenarios the appropriate set of `pssh` boxes is not known when the initialization segment is created.
```

Comment:
Nit: There are several places here where a dash is replaced with a comma. I would say this change makes the sentence structure less natural, at least to me.

Reply:
Keep dash instead of comma